canonical-web-and-design/ubuntu.com
# Add new 16-04 pages

## Done

- Add new pages at `/16-04` and `/16-04/gcp`
- Update release schedule chart to have the option of a highlighted row
- Separate the Kernel support schedule chart into it's own file and have the option of showing a particular tab on page load.

**Known bug**: page scrolls to bottom on page load. Trying to fix but ready for design review

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/16-04 and http://0.0.0.0:8001/16-04/gcp
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the pages against [16-04 copy doc](https://docs.google.com/document/d/1wclofqY9b3v_M9igpkYn6or5uXx0B_Xb_nVVzGBbOLY/edit#heading=h.wj8c1jlzgaj3) and [GCP copy doc](https://docs.google.com/document/d/14MeGfBfAiNxnKgJpTXhTEQpKMDAwoNp7Dqh0uqI7Dyc/edit)

## Issue / Card

Fixes [#3690](https://github.com/canonical-web-and-design/web-squad/issues/3690)
